src/Faker/Provider/Payment.php

```diff
@@ -211,9 +211,10 @@ public function creditCardDetails($valid = true)
      * @param  integer $length      total length without country code and 2 check digits
      * @return string
      */
-    protected static function iban($countryCode, $prefix = '', $length = null)
+    public static function iban($countryCode, $prefix = '', $length = null)
     {
-        $countryCode = strtoupper($countryCode);
+        $countryCode = is_null($countryCode) ? array_rand(self::$ibanFormats) : strtoupper($countryCode);
```

Comment:
Right, I've missed that. BTW, maybe [self::randomKey()](https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/Base.php#L198) should be used here? @fzaninotto often comments that `mt_rand` gives better randomness than lots of built-in shuffling PHP functions.